src/lib/isRgbColor.js

```diff
@@ -1,12 +1,34 @@
+/* eslint-disable prefer-rest-params */
 import assertString from './util/assertString';
 
 const rgbColor = /^rgb\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){2}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\)$/;
 const rgbaColor = /^rgba\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)$/;
 const rgbColorPercent = /^rgb\((([0-9]%|[1-9][0-9]%|100%),){2}([0-9]%|[1-9][0-9]%|100%)\)/;
 const rgbaColorPercent = /^rgba\((([0-9]%|[1-9][0-9]%|100%),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)/;
+const startsWithRgb = /^rgba?/;
 
-export default function isRgbColor(str, includePercentValues = true) {
+export default function isRgbColor(str, options) {
   assertString(str);
+  // default options to true
+  let strict = true;
+  let includePercentValues = true;
+  if (typeof options !== 'object') {
+    if (arguments.length >= 2) {
+      includePercentValues = arguments[1];
+    }
+  } else {
+    strict = options.strict !== undefined ? options.strict : true;
+    includePercentValues = options.includePercentValues !== undefined ?
+      options.includePercentValues : true;
+  }
+
+  if (!strict) {
+    // make sure it starts with continous rgba? without spaces before stripping
+    if (!startsWithRgb.test(str)) {
+      return false;
+    }
+    str = str.replace(/\s/g, '');
```

Comment:
A comment here is nice. We shouldn't expect people to know regex by heart.

```suggestion
    }
    // strip all whitespace
    str = str.replace(/\s/g, '');
```